boards/spark-core/doc.txt

```diff
@@ -0,0 +1,121 @@
+pause/*2
```

Comment:
bug ?

Reply:
I was suffering [this bug](https://vi.stackexchange.com/questions/13971/vim-pasting-yank-register-when-opening-a-new-file) in my local vim.